e2e/tests/dump-restore-test.rs

```diff
@@ -155,21 +253,21 @@ fn dump_db(container: &Container<Cli, GenericImage>, db: &str, username: &str) -
         .args(["-d", db])
         .arg("-v")
         .args(["-F", "p"])
-        .args(["-T", "_prom_catalog.\"default\""]) // todo: undecided atm
-        .args(["-T", "_prom_catalog.ids_epoch"]) // todo: Mat to look at
-        .args(["-T", "_prom_catalog.remote_commands"]) // todo: set seq to start at 1000, only dump records >= 1000
-        .args(["-T", "_ps_catalog.migration"]) // todo: should NOT be a config table. must restore at the same version at which we dumped
-        .args(["-T", "_ps_catalog.promscale_instance_information"]) // todo: should NOT be a config table
-        .args(["-T", "_ps_trace.tag_key"]) // todo: only dump records >= 1000
-        .args(["-T", "public.prom_installation_info"]) // todo: ???
-        .args(["-N", "_timescaledb_internal"])
-        .args(["-N", "_timescaledb_cache"])
-        .args(["-N", "_timescaledb_catalog"])
-        .args(["-N", "_timescaledb_config"])
-        .args(["-N", "_timescaledb_internal"])
-        .args(["-N", "timescaledb_experimental"])
-        .args(["-N", "timescaledb_information"])
-        .args(["-f", "/tmp/dump.sql"])
+        //.args(["-T", "_prom_catalog.\"default\""]) // todo: wrap in functions and only insert if not default value
```

Comment:
Do we need to keep these commented-out things around?

Reply:
I was leaving the comments here to remind me what I did. I'll fix up the commit messages accordingly and then nuke these comments.

Reply:
removed now